wikimedia/wikipedia-ios
# Table Views: Move to inset groups.

**Phabricator:** N/A

### Notes
* This change switches settings, libraries used, and the watchlist filtering view to use the new inset grouped style that was introduced with iOS 14. It also migrates notification filtering and inbox views to use inset groups as well.  This is a step towards modernizing the app to conform to modern iOS style guidelines.
* No compatibility check was added due to Wikipedia already requring iOS 15.4 and above.

### Test Steps
1. Open the app.
2. Go to each view that uses table cells, such as the following: Settings, Libraries Used, Filters, Inbox, and others.
3. Observe the modernized appearance of these design elements.

### Screenshots/Videos

![Simulator Screenshot - iPhone 15 Pro Max - 2024-08-29 at 06 40 57](https://github.com/user-attachments/assets/81abe020-fc04-435d-aa67-a3716bc2f7fa)
![Simulator Screenshot - iPhone 15 Pro Max - 2024-08-29 at 06 41 05](https://github.com/user-attachments/assets/0722154a-9656-43b0-8af4-26c3f283a916)
![Simulator Screenshot - iPhone 15 Pro Max - 2024-08-29 at 06 41 21](https://github.com/user-attachments/assets/c8a97d5c-f142-4c21-856e-9e9e397acab5)
![Simulator Screenshot - iPhone 15 Pro Max - 2024-08-29 at 06 41 34](https://github.com/user-attachments/assets/20061439-979d-4831-8f5a-23ebe3e99c84)
